packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -49,7 +49,11 @@ const SquadPage = ({ handle }: SourcePageProps): ReactElement => {
   const { user } = useContext(AuthContext);
   // Must be memoized to prevent refreshing the feed
   const queryVariables = useMemo(
-    () => ({ source: squadId, ranking: 'TIME' }),
+    () => ({
+      source: squadId,
+      ranking: 'TIME',
+      supportedTypes: ['share', 'article'],
```

Comment:
This one should actually, only accept `share` articles correct?
The regular source should then accept `article` only.

Reply:
removed 👍